### DISCUSSION TITLE
[Feature] approximate GPS with Google timeline

### DISCUSSION THREAD

**Discussion by arne182** [immich-app/immich#11843] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The ability to import the Google timeline export to have approximate locations for images that do not have the GPS in the exif tag.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1675/#3909/#6831

Please search for duplicates next time as requested in the template.
